templates/registration/logout.html

```diff
@@ -0,0 +1,6 @@
+{% extends "base.html" %}
+
+{% block content %}
+<h1>Logged out</h1>
+<p>You have been logged out. <a href="{% url "login" %}">Login again?</a></p>

```

Comment:
The URL for the login page should include the namespace 'taxi' to ensure it resolves correctly. Change `{% url "login" %}` to `{% url "taxi:login" %}`.